### ISSUE TITLE
Word - Function context.document.body.insertFileFromBase64 stopped working

### ISSUE THREAD

**Issue by emmanuelpineda** [OfficeDev/office-js#3040]
Without making any changes to the Add In code, the function context.document.body.insertFileFromBase64 stopped working about 2 weeks ago. I assume this is due to some Office API update.

## Your Environment

* Platform: Office on the web
* Host: Word
* Office version number: Office on the web
* Operating System: Any operating system since it is office on the web
* Browser: Edge (Chromium), Google Chrome
 
## Expected behavior
Calling the **context.document.body.insertFileFromBase64(base64File, Word.InsertLocation.replace)** function is expected to insert the content into the current document

## Current behavior
The **context.document.body.insertFileFromBase64(base64File, Word.InsertLocation.replace)** function throws an error and does not insert the content

## Steps to reproduce
Just call the function **context.document.body.insertFileFromBase64(base64String, Word.InsertLocation.replace)** from office on the web. Fails with any base 64 file string

## Context
This function is useful for our learning project so that the user does not start from scratch to create a document

## Useful logs
![image](https://user-images.githubusercontent.com/45979790/207478758-28e069d8-21db-4782-833e-809f36ed6942.png)

**Comment by wamaina** [comment]
Hello, if you go to File -> About on the document you should see some version information. Could you please provide this information as well?

**Comment by emmanuelpineda** [comment]
Thank you, here is the information requested:

![image](https://user-images.githubusercontent.com/45979790/207939090-ea123dd8-2903-436b-bcd8-ca903867547c.png)

**Comment by RuizhiSunMS** [comment]
@wamaina, thanks for the reporting and sorry for the late response since our team is encountering health issues. Will handle this issue as soon as our engs are back.

**Comment by xiaoyuMS** [comment]
Hi, @emmanuelpineda 
The error you provided does not show the error message from the JS API. Would you provide me with the relevant code to see why the "message" variable is null?

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS, I set this property (OfficeExtension.config.extendedErrorLogging = true) to try to get the error message, but still, the exception that occurs behind the Office API does not return it.

This is the detail of the error:

Image:
![image](https://user-images.githubusercontent.com/45979790/208594144-fcf5e178-7321-4f0d-8640-22918c10c816.png)

Error json file:
https://drive.google.com/file/d/1E5kjcPzJnJIdwexPH_BT0G_SImI7jokz/view?usp=share_link

Base 64 string:
https://drive.google.com/file/d/15-9i3_6WHNQosvp08T8XIEXsjyNN1Pol/view?usp=sharing

I am attaching the base 64 string (link) to reproduce the error: just try to insert the content.

This error is very strange as it didn't happen before for the same content.

Thank you.

**Comment by xiaoyuMS** [comment]
@emmanuelpineda , I can repro it with your base64 string and will investigate on it.

**Comment by emmanuelpineda** [comment]
Thank you @xiaoyuMS.

**Comment by hengchengfei** [comment]
@emmanuelpineda 
I tried linked base64 file，it‘s OK

```js
OfficeExtension.config.extendedErrorLogging = true;

  const base =
    "UEsDBBQABgAIAFVVc1JSWj6GTAEAABoFAAATAAgCW0NvbnRlbnRfVHlwZXNdLnhtbCCiBAIooAACAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAC1lE1OwzAQha9ieVslblkghJp2AWyhEr2A60xSC8e27Onf2VhwJK7AJGkjhEqDaLuJlMy8972xMv58/xhPt5VhawhRO5vxUTrkDKxyubZlxldYJHd8OhnPdx4io1YbM75E9PdCRLWESsbUebBUKVyoJNJrKIWX6k2WIG6Gw1uhnEWwmGDtwSfjRyjkyiB72tLnFhvARM4e2saalXHpvdFKItXF2uY/KMmekJKy6YlL7eOAGjgTRxFN6VfCQfhCJxF0DmwmAz7LitrExoVc5E6tKpKmp32OJHVFoRV0+trNB6cgRjriyqRdpZLaDnqDRNwZiJeP0fr+gQ+IpLhGgr1zf4YNLF6vFuObeX+SgsBzuTBw+RyddX8KpEWE9jk6O0hjc5JJrbPgfKTNDv8Y/LC6tTqhkT0E1D2/Xock77MnhPpWyCE/BhfNTTf5AlBLAwQUAAYACAAAACEAHpEat+8AAABOAgAACwAIAl9yZWxzLy5yZWxzIKIEAiigAAIAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAKySwWrDMAxA74P9g9G9UdrBGKNOL2PQ2xjZBwhbSUwT29hq1/79PNjYAl3pYUfL0tOT0HpznEZ14JRd8BqWVQ2KvQnW+V7DW/u8eACVhbylMXjWcOIMm+b2Zv3KI0kpyoOLWRWKzxoGkfiImM3AE+UqRPblpwtpIinP1GMks6OecVXX95h+M6CZMdXWakhbeweqPUW+hh26zhl+CmY/sZczLZCPwt6yXcRU6pO4Mo1qKfUsGmwwLyWckWKsChrwvNHqeqO/p8WJhSwJoQmJL/t8ZlwSWv7niuYZPzbvIVm0X+FvG5xdQfMBAAD//wMAUEsDBBQABgAIAFVVc1IEZyOzsggAAORQAAARAAAAd29yZC9kb2N1bWVudC54bWztXNlyozoafhWV56a7KmlLAgS4OjmDWXo76U51UqdrLmVQbLUBUQLH7XM17zCPMxfzLvMC8woj8JLFWWxixzA1uXFY9PEvn359Est//vmv97/9SmJwzWTORXrSQe9gB7A0FBFPhyedSXF1bHVAXtA0orFI2UlnxvLOb6fvp71IhJOEpQVQAGnem2bhSWdUFFmv283DEUto/i7hoRS5uCrehSLpiqsrHrLuVMioiyGC1X+ZFCHLc3U1l6bXNO8s4MJfm6FFkk5V4xJQ74YjKgv26wYDbQ1idO2utQ6EawApDzFah9K2hiLd0qo1IL0WkLJqDcmoh/SAc6QeEl5HMushaetIVj2kNTol6wQXGUvVwSshE1qoTTnsJlSOJ9mxAs5owQc85sVMYUKyhKE8HdewSLVaISRatDWC2U1ExGItWqII1b1l2lu0P161L03vzdsvfpYt5Cb+z5t4i+JQed6VLFaxEGk+4tmqhyd10dTB0RLk+iknrpO4s6pOaMPu8lh58uahvAHcxPxF/JN4bvnTiAhukJESYtViExPuXnNpSaJYeHPhWqG5FVy0YQFZAuA1ABKyDQv+EsNaYHTDmx5a4vANu8YSh6xweHQLp54xtwByth2EsbQjnyW3PMqGL6PtBykm2Q0afxnap5siNE23cxCS+zHK8pcZczGimapNSdj7NEyFpINYWaTIDBQfQZUBMCdE+QPmHANlIeiU8mUgoln5m4FpT4mf6Pu5Km8Q2qZm6E5nubPc5QYEq0aLXecPnOWxKzqJizsnI72XUUk/KToSw3GNvhvM96oRpSj3mou/yprsXFY/F8UsVmb3rml80rn85irx1T19312dIB8wrNp/LssTp73i9FLENBXAia5VxZxIlpfNiwpEzqEecNuExNPIM267HvawZm7qNoZ6H/ahc9dtHdlOoLv6bbfl4icQaZEr9FAxw6UxH0jeqTYLRQ+V3ISrPDt5yPkyLFXLB8JTeXXfXiVvXQQ93YedHVy1DKo74iEdikNF2LQDTRER3Y0wCZDWJ4a3gwirbrIkI1rucPO7u7bPwn3mvjALZT3p5RkN1VmZ4juT16xzqkEIkInyAlwUkrHiCNzJUjO48uqxOj0CtgU1vDPGQmJDx4abMhZi3fct0n++FK77etu5j06ab1YElm7UwSxj9Ma4n7G6SIg8mfuXGvoWGIZxrGsYgTcfGY3At2r4fLtJqnWbwH7wXKrvnfVc+bc0EkDdeCLVjwegOP3KwzH4EtM8Z+lDGRjIZQCeQ/pDRQGcq8LAIzUiPo81mmVMxmq+pTBHPC+EnFW1TvZKbSk/RfozxtfJXjqeu/rXohq+S/1zP3Eru161xxq+42q+47Snx57RiOciBa6YsjjeTe/dlG117L2U9JrFwOOShYps+zD4WUobe6B0ElYJaByjEfRd30KwPYx2EipBX0wKNTlqPp+dMBSTtKAP19pm2UoH86g+QdLXoiXpW4HR11skjVw1/waXYpoeKmSm4fg48MnuQlZOeF4hcn/RTYARBuc8ZeC7oNERWAUTAB3Cgwl0jZjQdPwWFUeleoFuEICwph8qakjHxEHOU1q3YVH7TBPOwAc+UEqp+WW6CRKJ7EEi/RxWCWicRMI20nxL22DFsil8/i4GPAX9mIbj5tPZidRBxTFJC37NgJPnPG+HXpKDMsINUEtm30UW1Fs0Lb0UUp29u7XiLQOm20Sz+9BvnVZCEIN///0fwIAQnFHVyZfruYuAHoFvX8EZ/gOgC2vnC2z3fHx0tRkcKq3Q06GBvB3qtVdK6xuC9PmSJTEt+2BlxFSTLtOw21RGBjPQZ6r+DtlehND/nnJ7brG5jl/FYJGCxmk3AyES2K7ZHkqfK3fKifBHJlOaRuzP5tO6vQouGy2j3AQVZ5s2stqk4n4XaSQOtuClGY7RJ/oOh6tXGu0xUtoti9lKvM3jCH642NlNn9lImO1cm3rewaiAdN/rm+2jAtQwMImhAwuTgwXP1olpGy0aI7/MYgrOwg+8DXc02zs6jpNwyJ+8Z/lqMi5w7PLGe3soeinGs4Mtb0Df9Vxste9WEE4jEMTvjsAZK6QA/Tgaqg2kH+tHwB/wfHIELkZ8MJnR4/GkXPQ4ZJCJ4xrYsFoX5DdQf6vpBjlGpmEYhwof6nuB4z/5RFTDuvSX8r0v8DeaiGLtwcXmFe8mrDVYe1hrGM+qBDRupYEE2DN8p0WLZ+5IqqRxJUnGXNLmM/o1hdSz1Lb3QO2QloloHLM1R00OMN7hzZJ9M/tiFqVsdrCRDesORnCH94tfSRiYOviS8uGoAM41SydMia0qkEfg68UPgMuXCJxJ2f9iTg8VXB36GjQ9vXXBLVVXeYfHOOizOQT2IQosrT192Z3Igufgg5D5mDd/kGqC7EJwH4PTsMpAzdFpR6sAfcNzUIvI6wz4kPIYuCIezJpP3gMuVUFiwOAFjy6HZYQ3XqnaDR1904Wav1/FX1sCMVoUMasVBkPTArTxC70GcXzf8vc7ka8bBssC55zJRDUBnlSkVppmHpkj8MMBwLYQMnc3GG8XORhg2/Tt/daz2noFQ7J4JAWTHT6Bvejom8o9w7dNs02zaieNJJuCs/Biytj9WUgjq34jJAvag2ShSZjPk/AKc+rteI0tB5q2gdvD69/pVLI0ZOAzTYf/J/VGpMZ7IHX8U8V/pyp82y9ZBIYT2IFbh7p7+pLFrr8L8JmXD9eCLzx5iDdP0Xb/ts1fZFf2pfSRZx4Pa9/PZMyTrYS4RzQNWg/Rb3XkNv1caFo+eUw0zVd0ctXpz+UDF6kuPrz4Ux2alit1WK/k/kj9b1jz/4XkLFWXz4RUEyFeLCCz4RktEQuRla84zc+V5WrZzeZAFIVIbrZjdnXr6IjRiKkkmLjavBKiuLU5nBTVJlxcT01oygwsnuGpTqr2RyL8IMvP+PTKxwTPeRGOyhfV4DIRc9erf+df9enefJ3w9L9QSwMEFAAGAAgAfY1zUr2lXNeHAQAAqgkAABwACAF3b3JkL19yZWxzL2RvY3VtZW50LnhtbC5yZWxzIKIEASigAAEAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAxZZNTsMwEIWvEnlP3fJTfkRaFrAogg10wXbqTlI3tieyp5SejQVH4gqYotJGFCldRFnOZPz86b1Rks/3j+vhmzXJK/qgyaWi1+mKBJ2iqXZ5KhacHV2I4eD6CQ1wnAgzXYYkHnEhFTPm8krKoGZoIXSoRBefZOQtcCx9LktQBeQoj7vdvvS7GqKqmYxXJdZRpCzTCm9JLSw63iMsZ1HJG+0KkYzB58ipsKAN01WxAktMN0wGHHUU2c3II03j7XdvjN6BEclomgo/ml6IRDbGmZHjMUwMbjl/W52o9kvRO2kSY4mTZ2SOcYctyE6zgtIoyf/B8SQOuhx9/ejO2yE188hZn7J33CRm+JNr2Bdqswy8MrhLsK6r+91OVPNcTwK5+mH12+EEq8IS0eHqgL1qyVOraInG1Ac9a4dT5eRDoQ8wtNsOqCsMhIAHbOlpS45CoT3Ux7xsEpPj2Z3P6rr8afaqb55GzVKLwGRfvu/bkGx6jw9SM9otzD2E6Cwo1q+4ZpKVP67BF1BLAwQUAAYACAAAACEA0xMIQyIGAACLGgAAFQAAAHdvcmQvdGhlbWUvdGhlbWUxLnhtbOxZTYsbNxi+F/ofxNwdf834Y4k32GM7abObhOwmJUd5Rp5RrBkZSd5dEwIlORYKpWnpoYHeeihtAwn0kv6abVPaFPIXKmk89siWWdpsYClZw1ofz/vq0ftKjzSey1dOEgKOEOOYph2neqniAJQGNMRp1HHuHA5LLQdwAdMQEpqijjNH3Lmy++EHl+GOiFGCgLRP+Q7sOLEQ051ymQeyGfJLdIpS2TemLIFCVllUDhk8ln4TUq5VKo1yAnHqgBQm0u3N8RgHCBwql85u7nxA5L9UcNUQEHagXCPDQmPDSVV98Tn3CQNHkHQcOU5Ijw/RiXAAgVzIjo5T0X9OefdyeWlExBbbgt1Q/y3sFgbhpKbtWDRaGrqu5za6S/8aQMQmbtAcNAaNpT8NgEEgZ5pxMX02a767wBZAWdHiu9/s16sGvuC/voHveupj4DUoK7ob+OHQX8WwAMqK3gbe67V7fdO/BmXFxga+Wen23aaB16CY4HSyga54jbqfz3YJGVNyzQpve+6wWVvAV6hyYXVl9qnYttYSeJ+yoQTo5EKBUyDmUzSGgcT5kOARw2APR7FceFOYUi6bK7XKsFKX/9XH1SWdUbiDYME6awr4RpPiA3jA8FR0nI+lV6cAefPyxzcvn4PTRy9OH/1y+vjx6aOfLVbXYBoVrV5//8XfTz8Ffz3/7vWTr+x4XsT//tNnv/36pR0oisBXXz/748WzV998/ucPTyzwLoOjIvwQJ4iDG+gY3KaJnJhlADRi/87iMIa4aNFNIw5TqGws6IGIDfSNOSTQgushM4J3mZQJG/Dq7L5B+CBmM4EtwOtxYgD3KSU9yqxzuq7GKkZhlkb2wdmsiLsN4ZFtbH8tv4PZVK53bHPpx8igeYvIlMMIpUgA1UcnCFnM7mFsxHUfB4xyOhbgHgY9iK0hOcQjYzWtjK7hROZlbiMo823EZv8u6FFic99HRyZS7gpIbC4RMcJ4Fc4ETKyMYUKKyD0oYhvJgzkLjIBzITMdIULBIESc22xusrlB97qUF3va98k8MZFM4IkNuQcpLSL7dOLHMJlaOeM0LmI/4hO5RCG4RYWVBDV3iKrLPMB0a7rvYmSk++y9fUcqq32BqJ4Zs20JRM39OCdjiLTz8pqeJzg9U9zXZN17t7IuhfTVt0/tunshBb3LsHVHrcv4Nty6ePuUhfjia3cfztJbSG4XC/S9dL+X7v+9dG/bz+cv2CuN1pf4/Kqu3SRb7+1jTMiBmBO0x7W6czm9cCgbdUUbLR8TprEsLoYzcBGDugwYFZ9gER/EcCqHqeoRIr5wHXEwpVyeD7rZ6lt1kFmyT8OstVrNn0ylARSrdnm+5O3yNBJZa6O5egRbute1SD8q5wSU7b8hURjMJFG3kGjmjWeQ0DM7FxZtC4uWcr+Vhf5aZEXuPwDVjxqemzGS6w0SFKo8ZfZ5ds8909uCaU67ZpleW3E9n0wbJArLzSRRWIYxDNF68znnur1KqUFPhWKTRrP1LnKtRGRNG0hq1sCx3HN1T7oJ4LTjjOXNUBaTqfTHlW5CEqUdJxCLQP8XZZkyLvqQxxlMd2XzT7BADBCcyLVeTANJV9yqtaaa4wUl165cvMjpr2KS0XiMArGlZVWVfZkTa+9bglWFziTpgzg8BiMyY7ehDJTXrKoAhpiLZTRDzAqLexXFNblabEXjF7PVFoVkGsPFiVIU8wyuy0s6hXlopuuzMuuLyYwilaS3PnXPNlIdBdHccoCoU9OuH+/ukC+wWum+wSqT7nWta+dat+2UePsDoUBtNZhBTTG2UFu1mtTO8UJQGG65NLedEed9GqyvWnVA5PdKXdt4NUFH9+XK78vr6owIrqmiE/mM4Oc/KmdKoFtzdTkRYMZwx3lQ8bquX/P8UqXlDUpu3a2UWl63Xup6Xr068KqVfq/2UAZFxEnVy8YeyucZMl+8edHtG29fkvyafSmgSZnqe3BZG+u3L9Xa9rcvAMvIPGjUhu16u9cotevdYcnt91qltt/olfoNv9kf9n2v1R4+dMCRBrvduu82Bq1So+r7JbdRUfRb7VLTrdW6brPbGrjdh4tYy5nn33l4Na/dfwAAAP//AwBQSwMEFAAGAAgAVVVzUgTkR1NGBAAAoAwAABEAAAB3b3JkL3NldHRpbmdzLnhtbLVX23LjNgz9FY2eq+hqx/asd8fXJjtxdydOp8+USNlsKFJDUnbcTr+sD/2k/kJBSbTsjbuTpM2TKRzgAAJAQP77z78+fHoqmLMjUlHBx254FbgO4ZnAlG/GbqVzb+A6SiOOEROcjN0DUe6njx/2I0W0BiXlAAFXo30ZJmN3q3U58n2VbUmB1FVBMymUyPVVJgpf5DnNiL8XEvtREAb1qZQiI0oB01yiPfy4LWGRPaMTJeEA5kIWSMOj3PgFko9V6QF9iTRNKaP6AORB39IIeAvJRy2Fd4zImIyaiNofayFf4rcxmYusKgjXtUdfEgYxCK62tFTH13grG4BbS7L73kvsCmb19mHwgtc1aT9avCS88zoVzAZIeef4TcU/jbz3OoLoGUE/I0+v4xi0HD5YnvJQ/Dqe/pGH4hOetwVzQqDI6yh6Ng51KLo3UuwlNW6gO5pKJA+nBS6y0e2GC4lSBuFAoR2olVNH5zTJMj9Ok3/HjAHXjActUfZ4T3bUDBbl7Ec7BHHkiCm4aL7RwCRHFdMPKF1rUVqN6yho8WyLgEMTuS5RBn03E1xLwawiFj8JPYN7L6EtrUk9BrrTuplRYMJRAdGfjYmVwBDKflRJ+vI0u9Z92Dvz+a0nASNVUkweTNrW+sDIEsJf09/IhOPPldIUKOtp8R9C+G4EhBvXX6DSD4eSLAnSFWTqvbzV1VgyWq6olELecgyD7P280TwnEjxQpMkKmohKsa9TfUMQhmX2Xo4rRX4Bbbhr8YNp8KnQWhQ3h3IL6f4/qumfNjEsZazs4V4IfdSd9JNkGkzbWA1soSCIF3E4SS5Cs2U/Ci9D834cB4NL0GwezaP4+gIUDhdROF0uL0H/HmE4C4N5sgguQFHQiweDYXQBShbxYDpdLGyW2twUI7Mnv0p7MrfMKRqTGSpSSZGzqjepb1RS+Til3CqkBCYgOYPWVWpRz2sRVSDGllBuiwQtgKkq5yRvHtgKyU3HbXXkZTEMv89Hvgw6mcgfpajKFt5LVDZ3yOqESWJtKdd3tLCAqtL10Y7D8D7BKo6/7GSTsi5TMJyhKeuBdIfq5q6VCfd+Xpt2JEjpiaJo7P6KvM9fjSilGHoaSW89sfeDybXpbbJCZdlckXQTjl1GN1sdGhsNTxg+z+qHdBO1WFRjUYPVDygzCQDt9tDJIis70YutLO5kiZUlnaxnZb1O1reyvpFtYS5KRvkj3FZ7NPJcMCb2BN90+DNRmwW1RSWZN4sMulE0gnazKWc3Ik8aMouphq/okuICwSdKGET92r5VZ+ggKn2mbDCjXZ5TYKTRcVKcWdd34ptozIrNKDTv+lCk3eK8amNnVMF8K2HJaiEt+EMDhskIi+zWrPakvX+DXjicz+IW79XrWddDEKp/T/IpUgRb0Br3GuPfl4NkEfaGQ+96MOt7yXS29Ab9SeIFw2U0iePJYhKEf9irbf9ZfPwHUEsDBBQABgAIAFVVc1JBwdZ0iwsAALh1AAAPAAAAd29yZC9zdHlsZXMueG1svZ3dVts6FsdfxStXcy5oSAIBug7nLKDtwDql5TR0eq3YCtFgSxnZLmVebS7mkeYVjizbicO2XG95D1eQj/3Tx19/Sdvxx//+899ff/+RxMF3rlOh5Plo8uZwFHAZqkjIh/NRnq0OTkdBmjEZsVhJfj565uno999+fXqbZs8xTwMTLtO3T5vJ0flonWWbt+NxGq55wtI3iQi1StUqexOqZKxWKxHy8ZPS0Xh6ODm0/220CnmamsLeafZk/owqYBICnNpwaT5cKZ2wzLzUD+OE6cd8c2DwG5aJpYhF9mzgh/Mao/tQypq9U2GecJnZ+LHmsSEqma7FJq1pT31o+81K4pKXMCG3GL++SuId4BgHmALAPOQ/cIzTijE2kU2OiHCc+ZYjogbHrzINQMpxiOO6HulzUrQoCd/ePEil2TI2JKNRYLo5sOCgbGfxJyi7LigG/KiwQaTCd3zF8jhLi5f6Tlcvq1f2zwclszR4esvSUIh7UzlTQiJMYdcXMhUj88m6+Kf1E87S7CIVrPXDMM0ab1+KyLw7tu78t/n0O4vPR9Pp9q2rFLwZM/lQv8nlwddFs8jGW0uDPh8xfbC4sJHjqnHjl03evHxli96wUNiC2CrjxpST+WFBjUUxpUyPz+oXX/Ki81meqbqUTVVKkzsG3W7Maqy7KKck8ylffVThI48WmfngfGQLM29+vbnTQmkzS5yPzs6qNxc8EdciirhsfFGuRcS/rbn8mvJo9/6fH6zTqzdClUvz/+xkbodCnEbvf4R8U8wb5lPJCmE+FQFx8e1c7Aq34f+qYZNajDbAmrNiMg4mLxlneMa0lZE2OqAs5UXrJ/iSZq9W0tGrlXT8aiXNX62kk1cr6fTVSjr7v5ckZGTWgElf7M9AUyrQjAp0RAU6pgLNqUAnVKBTKlDv4ekGZSqEC8SMCAxWDSowWCSowGBNoAKDJYAKDGZ8KjCY4KnAYD4/pOGC2ZuiwuUuLLgxfpPZcNxKqUyqjAcZ/0GAY9LAbLpJBCxWQq5p2knBKee5an0ejguZfQ0GSu/Fpu86nxXJYKBWwUo85Jqnw6vO5Xceqw0PWBQZICVR8yzXknBwa77imsuQk45wQmqRMQYyT5YUY3TDHuhgXEbUXVgjaWaI7cg2ufa68I+gGN0JC7WiWAYY3WTxUaQE/VVQgss8jjkV7BPRULMwggzCcggSCMshyB8shyB9aChH1k0Vjqq3KhxVp1U4qr4rBypZ31U4qr6rcFR9V+EI+u5eZDF/uUWZIA78XcUqJZkBF+JBMrM3IFiEqmOuwR3T7EGzzTooDm2DVg4v6FJFz8E9yVK3RZFt/+1IuTINFzIn6NQ9HJnPtkAqp22BVF7bAgncdmv20sUG7poo81nky6zVwIjsYcHivNz0EhiPZQQjbWeFD0KndIZo51IM5U/Flveaai+4qydB1XYwAoe9nKRoK1gxKeoZq/CRaGK+ft5wbXK4x+GoDyqO1ROPCJGLTKtyzDX9P5329//7ZLNmqUgBA7EJqM9KCG7ZZnib7mImJJF67w8SJuKAcHNxfX/7MbhXmyItLTqHiHipskwldNDqWOLfvvHlL0RVvDBps3ymavAF1aElS7sSFCtPiVIRFcpsRIUUNGurBf7Bn5eK6YgId6d5eVJQxqmQC5ZsYjKbmYnyyUxHFHslC/wH06I4pkTmr3saWuPIY5ov/8lDgqnvkwpojip9zjN7DNNuhwmypj0ewQ5ij0ewe7CamiWjGMgU7d3jEbR3j0fW3quYpakI6RpcA8laXAPJm0yQKlZAFSu9ymPCTqyJdL1YE+m6UcV5IlPSRlsgZZstkLzJlCPHAikO6Vng37WI6BSxNDI5LI1MC0sjE8LSaFUgOCmoQSM4M6hBIzg9qKRRbQ4aNLLxRrsxoPrpqEEjG2+WRjbeLI1svFka2XibvQv4amU2yoTrToNJNvYaTMLVR2Y82SjN9DMV833MHxjFUdYSd6fVqri2RMnytHKSHW++zEh35CWPTOpvfElXuQJGWjOKw6osjpWiOjS3W4Vs6Itz6X4WZy80GV6Ju5iFfK3iiGtXszoz7EV5zcjLFtiK9Dt2+lE8rLNgsd7+eNDkFBeg/Cy0TvL34noU2dbz82lX3C2PRJ7UdYWn8s5niGhwvu78qEf0bpuxF3rcNxSWOu8RuttM74We9A2FpZ72DQVnH887zfGO6cfWEXHSOZK2SaFjHJ50jqdtdGvBnUNqG9o2Gk86x9OecYKLMCx+gIAi9XSQG9DTSm4AylNuDMpcbkx/l7kZnXb7wr+LtP2Q909+at+erwEWhKP+8+mfucrAD+JTxGVoN2ZzJVMetIJmiF/F9uYdd2f2n4DcjP4zkZvRf0pyM/rNTc543CTlxvSfrdyM/tOWm4Gfv+BKgZy/IAA5f0GA1/wFMV7z15BdgpvRf7vgZuBtCxl42w7ZSbgZONuCeD/bQgzetpCBty1k4G0Ld2lI20IA0rYQ4GVbiPGyLcTgbQsZeNtCBt62kIG3LWTgbeubCTjj/WwLMXjbQgbetpCBty24AhNrWwhA2hYCvGwLMV62hRi8bSEDb1vIwNsWMvC2hQy8bSEDZ1sQ72dbiMHbFjLwtoUMvG3B9c1Y20IA0rYQ4GVbiPGyLcTgbQsZeNtCBt62kIG3LWTgbQsZONuCeD/bQgzetpCBty1k4G0L7h6AtS0EIG0LAV62hRgv20IM3raQgbctZOBtCxl420IG3raQgbMtiPezLcTgbQsZeNtCRudIrX4RdV0SMPE4iuq8vADxE1lVrS/Ny9T3DsoiWHW93DDEtROXSj0GrZdQzmYIiljGQtkD38+AQ3H6xeer5sVJfnct6duY6uIN+xstOCB61DsUHJQ56hz8zVCQGB51jvlmKNicHnXOyM1QsEAedU7E1qT1eTFmmQLRndNOI3riiO+cwhvxsKM7J+5GJOznzum6EQm7uXOSbkQeB8WM/TL8uG9nzbdnvwJE58hsIE7ciM4RCiVz/mzQXzs3oreIbkRvNd0InKxOjoe+bhZeaDfLU3HoObTiA2zrRqAVhwg/xQFngOKQ5a84ZHkqDudKtOIQgVZ8wIztRvgpDjgDFIcsf8Uhy1NxuMahFYcItOIQgVZ86GLt5AxQHLL8FYcsT8XhDhCtOESgFYcItOIQ4ac44AxQHLL8FYcsT8VBdo1XHCLQikMEWnGI8FMccAYoDln+ikNWp+L2KMyA9KoRj9ynNSKRi3UjEjljNyJ90qtGuG961UD4pldQMs/0qqmdZ3rVFNEzvWqq6ZleAVk906tWfT3Tq1ahPdMrt+LI9KpN8QG29Uyv2hRHpldOxZHpVafiyPSqU3FkeuVWHJletSmOTK/aFB8wY3umV07FkelVp+LI9KpTcWR65VYcmV61KY5Mr9oUR6ZXbYoPXaw906tOxZHpVafiyPTKrTgyvWpTHJletSmOTK/aFEemV07FkelVp+LI9KpTcWR65VYcmV61KY5Mr9oUR6ZXbYoj0yun4sj0qlNxZHrVqTgyvbo1MYLilliLhOksILyb3jVL1xkjuIXjV6l5quLvPAqIW/sR19Dx095Txwq4fSSiCchMvxU3lG9cc2U/uomazwOLynvpFrwiuKhKUD2Jrf6SrXL163BVpgXBwsK1KS2sb+dVF1bdrnd79Vd9s96Ool13+LVV2Y3E+ut1v+46rfrmXp911z0rBv9eva0dED1V3yzMUcmzs961NHVaxuVD68w/NzIyjKfq2SZlbaMfbLT95hWP41tWfl1tOr4b81VWfjw5PG37wrK8S6GboO0E7kaM9ys03rbE3fHl0w7qO6vuRmh9sWlnv1eXpA7vcnf92h10//nqFNSneiJN2ZXMkD/LVjPJ4naobR8Uzxz8wqPitoK8/2DRafFYyBJ3eDh7P5tc1Oc1uB59OD1sPvrwZF6/cD/6sOqd+r/0t78AUEsDBBQABgAIAAAAIQBAwf/xOQEAACMDAAAUAAAAd29yZC93ZWJTZXR0aW5ncy54bWyc0l9PwjAQAPB3E7/D0nfoILKQhUFiDMYXY6J+gNLdoLHtLb3iwE/vbQJieGG+9P/9ctd2ttg5m3xCIIO+EKNhKhLwGkvj14V4f1sOpiKhqHypLHooxB5ILOa3N7Mmb2D1CjHySUpY8ZQ7XYhNjHUuJekNOEVDrMHzZoXBqcjTsJZOhY9tPdDoahXNylgT93Kcppk4MOEaBavKaHhAvXXgYxcvA1gW0dPG1HTUmmu0BkNZB9RAxPU4++M5ZfyJGd1dQM7ogIRVHHIxh4w6isNHaTdy9heY9APGF0CmYdfPmB4MyZHnjin7OdnJMeWZ879kzgCCfsTkmAftXVuR0/nT2mNQK8sSv1HC15x0cNtytm3XDfn4nH8t1tE48wVLDPcBG4Ig22VlLTYvz488kX++9vwbAAD//wMAUEsDBBQABgAIAFVVc1LKQXTdiAIAAKYJAAASAAAAd29yZC9mb250VGFibGUueG1s1ZVLbtswEIavQnCfiJafMaIEjhMVBZos2l6ApiiLqEgKJB3H22TfdRftEYouWqCLBuhhDGTrK3T0cl04bqIADVAJkOkZzqfh75/06sfPw+MrmaJLbqzQKsCtfYIRV0xHQk0DPHPx3gAj66iKaKoVD/CCW3x8dDgfxlo5i6Ba2aFkAU6cy4aeZ1nCJbX7OuMKkrE2kjr4aqaepObdLNtjWmbUiYlIhVt4PiE9XGHMYyg6jgXjp5rNJFeuqPcMT4GolU1EZmva/DG0uTZRZjTj1sKKZVryJBVqjWl1tkBSMKOtjt0+LKbqqEBBeYsUI5n+BnSbAfwtQI/xq2aMQcXwoHKTI6JmnN6aI6INztOa2QBY3gzRrfuwC5mvSLLhy6nShk5SIMFvhEBmVIBRuc78A5XS4dqvaD5UVELBmKZiYkSRyKjSlrcgd0nTABOfhKQLz/zukHb+xMjLZ7KEGsvdeiap4jGVIl3UYTsX1laZTDiW1IlLakTeb5WzYgqZmZ2QAJ8RuPwwxGWkFeAOBEbjdcTPX1dcrSrSXkdIHmEFp5xxEFaR1uYceKlXyrAlxwgaS3eIcUJ6IIO/vp9HDP9sU4wxRPqD7smWGAcPixE2FeOtkNyiCz5Hr7WkaocsPsjSBp90Cq+0G8piCvITZBlty9JpP4ss52/QC+0SwQpBaOouIFy3vbr9vLr9ipbX35bX35c3N8vrL/Xq7jXUAenDqEsGuw01uF85qSNu7pUuFlc8etTe6o3G/fA03LTToBDFf0A3+HNovLeqowa9EtPE7Txw2s984IxIucn+PHB80t/eY+QfHDhgpnOhWKL/Yqa7D+/vPn7aaSO/sFGnstHODfjf2qga2KNfUEsDBBQABgAIAFVVc1JflJHYVQEAAH4CAAARAAgBZG9jUHJvcHMvY29yZS54bWwgogQBKKAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACNUt1OwjAUfpWl91v3QwCXMhIlXkliIlHjXW0PUNnapT0weDYvfCRfwW2MCeqFvTrt99Nzvvbz/YNN90Xu7cA6ZfSEREFIPNDCSKVXE7LFpT8m04wJY+HemhIsKnBerdEulWJC1ohlSmm5tXlg7IpKQSGHAjQ6GgURJT0XwRbuT0GL9My9Uz2rqqqgSlpeHIYRfZ7fPYg1FNxX2iHXAjpVr3At7IK6VV0jS2MLjq51KLnY8BU0TkNaAHLJkdNmMr/sRyMZkyJFhTl4tK3d9vUNBHY7YYGjsdmMawW59wg5d8IwegaxDRwqY6XrJBKcsKrEOuDmpBbgvM53qUBeH34Z/YCZhZ1q3iZLGO1r1sV5vBOkV4eW4qGECTkhT8nNbHFLsjiMIz9M/OhqEY3SeJiGYRAe10vT9oXPt3HRtfBv50GcDpIgGVyN48HozPlklDF6+YmyL1BLAwQUAAYACABVVXNSf5FPfzsBAACnAgAAEAAIAWRvY1Byb3BzL2FwcC54bWwgogQBKKAAAQAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACtkk1OwzAQha9ied8kZVGhqkmF0gULQEgtZW3sSWPhP9nT0pyNBUfiCtgpbYLYIbzze988z4z8+f6xWB61IgfwQVpT0mlWUAKGWyHNrqR7bCbXlARkRjBlDZS0g0CX1YK5+aO3DjxKCCRmmFDSFtHN8zzwFjQLWbRNdBrrNcN49bvcNo3ksLJ8r8FgflUUsxyOCEaAmLhLID0lzg/411BheeovbDedG/KY+88m+y1sQDvFEKqHlKAyYVEv8rGRqBvnlOQM45Kre8m9DbZB8my9IPFlgi2QN3jp68ZoKo19rIHvvcSuKnpirCRizZmCOvZVNUwF6JlBS0RttWOmI/mJb5kHEVPG/EVLxG1cmlfSvIa6ZWYHYkT+9r4H3J4+UTWdZUU852HOcqLuUtmT29hVWsyQ+VPvpeF3VV9QSwMEFAAAAAgAfY1zUsVqx45+AAAAqAAAABIAHABjdXN0b21YTUwvaXRlbS54bWwgohgAKKAUAAAAAAAAAAAAAAAAAAAAAAAAAAAAXY1LCsMgFACvIu6tihoSeJpdIN22PYC/FiHVohJ6/GbZdjvMMDC/nxvaY22pZI35iWEUsy8h5YfGt+tCRjwbONv2itX6nva45ntBR5Wbxt8cG7h0W3usS9qigTUYPlgnBPOEKy+JDHYgTnFGJs5lEMxNSo5ADxHoT0r/f+YDUEsBAi0AFAAGAAgArlRzUlJaPoZMAQAAGgUAABMAAAAAAAAAAAAAAAAAAAAAAFtDb250ZW50X1R5cGVzXS54bWxQSwECLQAUAAYACAAAACEAHpEat+8AAABOAgAACwAAAAAAAAAAAAAAAACFAwAAX3JlbHMvLnJlbHNQSwECLQAUAAYACABVVXNSBGcjs7IIAADkUAAAEQAAAAAAAAAAAAAAAAClBgAAd29yZC9kb2N1bWVudC54bWxQSwECLQAUAAYACAB9jXNSvaVc14cBAACqCQAAHAAAAAAAAAAAAAAAAACGDwAAd29yZC9fcmVscy9kb2N1bWVudC54bWwucmVsc1BLAQItABQABgAIAAAAIQDTEwhDIgYAAIsaAAAVAAAAAAAAAAAAAAAAAE8SAAB3b3JkL3RoZW1lL3RoZW1lMS54bWxQSwECLQAUAAYACABVVXNSBORHU0YEAACgDAAAEQAAAAAAAAAAAAAAAACkGAAAd29yZC9zZXR0aW5ncy54bWxQSwECLQAUAAYACABVVXNSQcHWdIsLAAC4dQAADwAAAAAAAAAAAAAAAAAZHQAAd29yZC9zdHlsZXMueG1sUEsBAi0AFAAGAAgAAAAhAEDB//E5AQAAIwMAABQAAAAAAAAAAAAAAAAA0SgAAHdvcmQvd2ViU2V0dGluZ3MueG1sUEsBAi0AFAAGAAgAVVVzUspBdN2IAgAApgkAABIAAAAAAAAAAAAAAAAAPCoAAHdvcmQvZm9udFRhYmxlLnhtbFBLAQItABQABgAIAFVVc1JflJHYVQEAAH4CAAARAAAAAAAAAAAAAAAAAPQsAABkb2NQcm9wcy9jb3JlLnhtbFBLAQItABQABgAIAFVVc1J/kU9/OwEAAKcCAAAQAAAAAAAAAAAAAAAAAIAvAABkb2NQcm9wcy9hcHAueG1sUEsBAi0AFAAAAAgAfY1zUsVqx45+AAAAqAAAABIAAAAAAAAAAAAAAAAA8TEAAGN1c3RvbVhNTC9pdGVtLnhtbFBLBQYAAAAADAAMAAEDAAC7MgAAAAA=";
  Word.run(async (context) => {
    const range = context.document.getSelection();
    range.insertFileFromBase64(base, "replace");
    await context.sync();
  }).catch((e) => {
    console.log(e); 
  });
```

![1](https://user-images.githubusercontent.com/3182964/209632415-f6c3b00d-e34f-40a3-9fba-96b464dcbf41.png)

**Comment by emmanuelpineda** [comment]
Hello @hengchengfei,
I just tried, but the error persists. Attached the error screen and the version of Office Web. The content (base 64 string) is the same.

![image](https://user-images.githubusercontent.com/45979790/209889035-53bced46-5009-49fa-bd94-fc7995800c02.png)

**Comment by emmanuelpineda** [comment]
How can we debug/fix this error? @xiaoyuMS was able to reproduce it last week:

![image](https://user-images.githubusercontent.com/45979790/209890267-2916a3f2-714b-442a-8099-a548580a41d4.png)

**Comment by hengchengfei** [comment]
> Hello @hengchengfei, I just tried, but the error persists. Attached the error screen and the version of Office Web. The content (base 64 string) is the same.
> 
> ![image](https://user-images.githubusercontent.com/45979790/209889035-53bced46-5009-49fa-bd94-fc7995800c02.png)

I tried in Windows Desktop rather than Office Online web.

![1](https://user-images.githubusercontent.com/3182964/209896236-6c535088-3fa8-443b-b94e-ed51c50b73cb.png)

**Comment by emmanuelpineda** [comment]
Yes, in Desktop Office it works fine, but in Online Office (web) it doesn't

**Comment by emmanuelpineda** [comment]
Hi @hengchengfei,

What is the estimated time to solve this issue?
I think we need to inform our clients because they are waiting for this.

Thank you.

**Comment by hengchengfei** [comment]
@emmanuelpineda 

I am also an Office developer，only develop in Desktop Office.

**Comment by xiaoyuMS** [comment]
Hi @emmanuelpineda, sorry for the late response due to the holidays.

We create a bug to track this regression. It is a complex issue which used to happen on those file hosts with WOPI protocol. Unfortunately, we do not have any timelines to share at this point. We will update here if there is any progress.

By the way, which file host are you using, SPO or onedrive or something else? The API currently works fine on onedrive.

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS and @hengchengfei, thank you for your help.

The user creates and opens a document on its OneDrive, opens our add in to execute an action that replaces/inserts the content of the document.

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS, Do you have any update?

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS,

We were doing tests and we detected that there is a new version of Office Online.

In this version the insertion of content does not work:
![image](https://user-images.githubusercontent.com/45979790/213250235-a8ba2486-8df2-48e0-b94d-f1bd5896f2d6.png)

In this version the insertion of content works correctly:
![image](https://user-images.githubusercontent.com/45979790/213250509-757b8555-3472-4851-a13c-94841a50853d.png)

It seems that the versions are loaded randomly based on the user's location. How can we force the latest version of Office to load?

Thank you.

**Comment by xiaoyuMS** [comment]
@emmanuelpineda Sorry for the late response, just come back from holidays.

Currently we have put this bug in our backlog, but we need to handle them according to priority.

I am not familiar with the build version of Word Online. Are you still facing the version that can't not work?

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS,

It still doesn't work, this is the version I tried:

![image](https://user-images.githubusercontent.com/45979790/215800947-af165e83-5d8c-4af1-b5a4-2ba9bae476b4.png)

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS,

It still doesn't work, this is the version I tried:

![image](https://user-images.githubusercontent.com/45979790/215800947-af165e83-5d8c-4af1-b5a4-2ba9bae476b4.png)

Is there a way to load the latest version? I have noticed that this version works well: 16.0.16127.41003

Thank you.

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS,

It still doesn't work, this is the version I tried:

![image](https://user-images.githubusercontent.com/45979790/215800947-af165e83-5d8c-4af1-b5a4-2ba9bae476b4.png)

Is there a way to load the latest version? I have noticed that this version works well: 16.0.16127.41003

Thank you.

**Comment by xiaoyuMS** [comment]
@emmanuelpineda 
As far as I know, there is no way to specify a version of Word Web, which is supposed to be the latest for users. Users may be located in different deployment rings with different versions.

According to a preliminary investigation, this is not a regression related to build versions but should be deeper bug in the API execution chain. 

I notice my current version works well. 
<img width="310" alt="image" src="https://user-images.githubusercontent.com/90585978/217177752-6ccda729-2beb-4900-bccd-72f3ab559afb.png">

**Comment by emmanuelpineda** [comment]
Thank you @xiaoyuMS,

I have checked the content loading, most of the cases have been fixed, except for one that throws me the following error:

![image](https://user-images.githubusercontent.com/45979790/217337999-a04c32dc-f997-4cf5-a7a2-199152e9099b.png)

Office Web Version:
![image](https://user-images.githubusercontent.com/45979790/217338903-26dd9704-027c-4ca8-b0bf-4172b064ef55.png)


Here you can download the base64 string as a text file: https://drive.google.com/file/d/1XFB5q_-BH06sXq0vkJ-q-eBE-J8WJ1k4/view?usp=sharing

Thank you for your help.

**Comment by xiaoyuMS** [comment]
@emmanuelpineda 
Weird, I got the same error message on inserting this base64 string. Can't repro the message "err is null". 
<img width="400" alt="image" src="https://user-images.githubusercontent.com/90585978/217420190-22c60cb8-5445-49cd-ba66-169c58f227a2.png">

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS,

Just to let you know, in previous versions of Word Online, content loading was already working in most cases of my product, today I did a test and it stopped working as it did at the beginning.

![image](https://user-images.githubusercontent.com/45979790/219454686-95f3f775-e9a8-481d-92c1-c9f4e29b5455.png)

![image](https://user-images.githubusercontent.com/45979790/219455930-39f1f340-4e2b-46c8-87a4-9ae8ed008fe5.png)

**Comment by veysby** [comment]
Any updates on this? - We've noticed this today with stack trace similar to the comments above. 

Environment: Office for the web
Build# 16.0.16413.41001

**Comment by xiaoyuMS** [comment]
Thanks for reaching us, @veysby
It is a bug related to insertFileFromBase64 but not a regression. I have created a bug VSO#7367799 in the backlog for internal tracking. We'll put it on the schedule based on priority. Unfortunately, we don't have anything to update right now.

**Comment by arghyadharmsft** [comment]
@xiaoyuMSL Hello Yu, Please update me on the status of fix when you have resolved the issue.

**Comment by emmanuelpineda** [comment]
Hello @xiaoyuMS,

Do you have any update please?

**Comment by dvelascozarate** [comment]
Hi @xiaoyuMS. I have the same problem when inserting content, do you have any update on the correction of this error? I have to provide dates to the company I work for, so at least an estimate would be helpful.

**Comment by xiaoyuMS** [comment]
Hi @dvelascozarate @emmanuelpineda @arghyadharmsft, we have checked in a fix for this unexpected exception, but it needs a few weeks to deploy. 

In addition, the execution time of insertFileFromBase64() may vary depending on the environment, so we add a timeout mechanism that returns a failure exception if it times out.

**Comment by xiaoyuMS** [comment]
Hi @dvelascozarate @emmanuelpineda @arghyadharmsft, the fix has been deployed to production yesterday, please have a try.

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS ,

I have this version:
![image](https://github.com/OfficeDev/office-js/assets/45979790/47c2428a-0cfe-4cf3-b045-b21839a90cb3)


Unfortunately it still doesn't work, after waiting for 1 minute or so it didn't insert the content and on the console I got this error:

![image](https://github.com/OfficeDev/office-js/assets/45979790/95834915-87b0-42c3-9fc9-244469a04934)

The content I tried to insert is a document with only 1 page (text only).

I'm still waiting for this bug to be fixed

**Comment by xiaoyuMS** [comment]
Hi @emmanuelpineda ,
There is a known limitation for those server-side APIs like insertFileFromBase64() **on Word Online**. Even if the insertion has happened on the file, it relies on the file host to synchronize the inserted content back. Sometimes it is slow, so we add the timeout mechanism to mitigate this issue.

It's a behavior that's been around for a long time, unfortunately, it seems like a regression on your part. We have noticed the flaw of this API and have put it on our list. However, due to limited resources, we don't have any concrete timeline to share yet.

**Comment by gasapu** [comment]
Hi @xiaoyuMS I am also facing same issue as @emmanuelpineda facing,
while using insertFileFromBase64() on Word Online.

error message - "Can't find the inserted content. Please refresh the page for the result." 
do we have any update on it.
Thank you!.

**Comment by lamotik** [comment]
Hi @xiaoyuMS,
We experiencing same issue with Word Online and insertFileFromBase64. Are there any possible workarounds on how to load document? This but is critical for us. When can we expect it to be fixed?

**Comment by xiaoyu03** [comment]
@gasapu, @lamotik Thank you for your patience and interest in this API. Unfortunately, we do not have a definite timeline for enhancing this API. In the meantime, you may want to try using the insertHtml() or other insertion APIs as alternatives.

**Comment by emmanuelpineda** [comment]
Hi @xiaoyuMS,

Since the insertFileFromBase64 method failed, I searched for alternatives, currently using insertHtml, as a temporary solution. I also tried using Office.context.document.setSelectedDataAsync but it still fails like insertFileFromBase64. Hopefully the API will be fixed soon because insertHtml is a very limited alternative for our product.

**Comment by lamotik** [comment]
@emmanuelpineda 
Thank you for reply, the thing is that we need not just body, but also header and footers insert, while insertHtml provides only body replacement. But as there are no other options, I guess it must be used.

@xiaoyu03 
Do you have some sort of priority list where we can see when this bug will be fixed?

**Comment by emmanuelpineda** [comment]
Hello @xiaoyu03,

Do you have any updates on the status of this issue, I would greatly appreciate your feedback, we are still waiting.

Thank you.

**Comment by emmanuelpineda** [comment]
Hello @xiaoyu03,

We have 2 add-ins (XperienceED and Jasperactive) published in the Microsoft store and we have customers waiting for a solution to this issue, is it possible to set a high priority so that it is resolved as soon as possible?

**Comment by xiaoyuMS** [comment]
Hi @emmanuelpineda, the good news is we are working on a fix for this reloading issue, I will get back to you once it is ready ~

**Comment by xiaoyuMS** [comment]
Hi @emmanuelpineda @lamotik @gasapu @dvelascozarate , the fix has been deployed to production, please have a try. The full deployment may take another 1-2 days.

**Comment by SirDavenport** [comment]
Hi @xiaoyuMS ,
Will this fix be deployed for test wopi as well?

**Comment by xiaoyuMS** [comment]
@SirDavenport I am not sure about it, have you tried it?

**Comment by lamotik** [comment]
> @SirDavenport I am not sure about it, have you tried it?

Unfortunately not working. Can wait few days more.

**Comment by SirDavenport** [comment]
Yeah seems like its working in sharepoint OfficeOnline, but not my test wopi instance.

**Comment by emmanuelpineda** [comment]
It seems to be working now, thank you very much!

**Comment by xiaoyu03** [comment]
Hi @SirDavenport  @lamotik We have deployed a fix for the WOPI issue, please have a try~

**Comment by emmanuelpineda** [comment]
Hello @xiaoyu03,

We have a new problem with the method: insertFileFromBase64, it does not throw any error but does not insert the content, it worked well in the past days and now it is not working. Please we need your help, we have a large number of users (students) who use the Jasperactive Add In and are stuck when starting this new school year.

**Comment by knopmickael** [comment]
@emmanuelpineda review your base64 string, the behavior was the same here some weeks ago, but the guys fixed it recently

The real problem was:
- if the docx used has only ONE page it starts the infinity loop when inserting it as base64 through the isnertFileFromB64 method, it does not throw any exception and doesnt import too (Word online - chrome browser)

Check this other issue, I detailed our working case (https://github.com/OfficeDev/office-js/issues/4682#issuecomment-2369243454)

**Comment by emmanuelpineda** [comment]
Thank you for the info @knopmickael!, I'll check the Word document

**Comment by xiaoyuMS** [comment]
Hi @emmanuelpineda, 

May I know if the API works for you now? If not, could you share some failed samples with us to repro the issue?

**Comment by SirDavenport** [comment]
Hi @xiaoyuMS ,
Actually now that you mention it, the insert does not always work for me either. Just tried to insert a relatively big word document into a content control and it briefly shows the "inserting" loading spinner, and then just doesn't do anything and these errors show up
![image](https://github.com/user-attachments/assets/5e613a29-cc3d-4cbf-af1b-878de6227952)

**Comment by xiaoyuMS** [comment]
@SirDavenport 

This looks like a new issue. Did it continue to happen for particular insertion? If so, is it possible to provide the repro steps with us?

**Comment by ranshul-microsoft** [comment]
Closing since there has been no response and the scope of the original issue was resolved as well. Please open a new issue if you continue to face this.
